Component/Pet/Pet.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+const StyledPet = styled.div`
```

Comment:
I suggest using semantic HTML elements like `<section>` instead of `<div>`

Reply:
I suggest the same. :)